### ISSUE TITLE
Segmentation Fault when importing

### ISSUE THREAD

**Issue by harper-carroll** [JuliaPy/SymPy.jl#250]
I am trying to use SymPy by running "Using SymPy" and I get the following error:

![image](https://user-images.githubusercontent.com/20361958/50494578-10cb3800-09f2-11e9-922e-a17ae948fef9.png)

Apparently this is an issue with SymPy itself. Is that correct?

**Comment by jverzani** [comment]
Hmm, I will need some more detail here (OS, result of `] add SymPy`, ... I can only guess something went wrong in the installation. As a first step, can you issue the command `using PyCall` successfully?

**Comment by tkoolen** [comment]
Running my tests on nightly results in

```julia
ERROR: Error while loading expression starting at /home/travis/.julia/packages/SymPy/Pp4za/src/SymPy.jl:148
caused by [exception 2]
Error while loading expression starting at /home/travis/.julia/packages/SymPy/Pp4za/src/specialfuns.jl:115
caused by [exception 1]
syntax: malformed "import" statement
Stacktrace:
 [1] eval at ./boot.jl:328 [inlined]
 [2] eval(::Expr) at /home/travis/.julia/packages/SymPy/Pp4za/src/specialfuns.jl:1
 [3] top-level scope at /home/travis/.julia/packages/SymPy/Pp4za/src/specialfuns.jl:117
 [4] include at ./boot.jl:326 [inlined]
 [5] include_relative(::Module, ::String) at ./loading.jl:1041
 [6] include at ./Base.jl:29 [inlined]
 [7] include(::String) at /home/travis/.julia/packages/SymPy/Pp4za/src/SymPy.jl:4
 [8] top-level scope at /home/travis/.julia/packages/SymPy/Pp4za/src/SymPy.jl:148
 [9] include at ./boot.jl:326 [inlined]
 [10] include_relative(::Module, ::String) at ./loading.jl:1041
 [11] include(::Module, ::String) at ./Base.jl:29
 [12] top-level scope at none:2
 [13] eval at ./boot.jl:328 [inlined]
 [14] eval(::Expr) at ./client.jl:444
 [15] top-level scope at ./none:3
```

so

https://github.com/JuliaPy/SymPy.jl/blob/3da7674336f271c06a38629ea0c00b0950603c92/src/specialfuns.jl#L117

is likely to be the culprit.

**Comment by tkf** [comment]
@tkoolen Do you have the latest SymPy.jl v0.8.4? I think that's what I solved in #252.

**Comment by tkoolen** [comment]
Ah, great, thanks for the fix. My tests were using a pinned, older version of SymPy.
